projects/truesight.md

```diff
@@ -0,0 +1,74 @@
+## Project Name <!-- Add your project name here with format "Project Name"-->
+TrueSight
+
+## Category 
+<!--developer tooling, application, wallet, infrastructure, etc-->
+Infrastructure
+
+## Project Description
+<!--Describe your project in a few sentences. -->
+TrueSight aims to be a decentralised oracle network built to provide moderation labels for user-generated content.
+
+Typically content moderation is done in a centralised and opaque manner and may be unfairly influenced by stakeholders of the application.
+By making over the blockchain applications are able to prove to users that moderation labels are generated by the impartial oracle network.
+By off-loading storage of both the user-generated content and the moderation labels on IPFS and only storing the content hashes for both on the blockchain gas costs are significantly reduced.
+
+## Project Status
+<!--brainstorming, fundraising, under development, beta, shipped, etc-->
+Prototype created as a submission to the Spring 2021 Chainlink Hackathon. Submission included a demo client application which makes requests to the protptye network. Project was awarded Runner Up. See https://devpost.com/software/truesight
+
+## Previews
+<!--Add some screenshots to give a preview of your product-->
+![image](https://user-images.githubusercontent.com/6655367/115218823-dc1da600-a0fe-11eb-85ec-705f594f7bd3.png)
+
+## Target Audience
+<!--Describe who will be your project's users-->
+Developers of social applications.
+
+## Github repo
+<!--Attach a link to your GitHub repo if it's OSS-->
+https://github.com/H0M13/TrueSight-explainer
+
+## Website
+<!--Link your website if available-->
+
+Explainer website:
+https://truesight.link/
+
+Demo application:
+https://truesite.link/
+
+## Team Info
+<!-- Introduce your amazing team - how many team members are working on this project and who are they?-->
+We're a team of good friends with a mix of skills between web development and data science.
+
+### Team Size  
+
+4
+
+### Team members  
+
+Matt Durkin - https://github.com/PickleyD
+
+- Creative full-stack web developer with 8 years professional experience.
+- Winner of the [2020 Chainlink Virtual Hackathon](https://chain.link/hackathon/2020).
+
+Sam Humby - https://github.com/Samalot 
+
+- Full-Stack Software Engineer, 4 years professional experience developing Frontend Frameworks and Backend microservices. 
+- https://samalot.dev/
+
+Harry Jackson - https://github.com/harryjjackson
+
+- Data Scientist specialising in manufacturing optimisation with a background in computer science.
+
+Charlie Stafford-Roberts - https://github.com/colossalchaz
+
+- Public health data analyst working for local government to support the covid-19 recovery effort.
+
+## How the community can engage
+Discord: https://discord.gg/UFvuED5D
```

Comment:
Hey! can you add a show-and-tell topic in the discussions forum and link it here? thanks!

https://github.com/ipfs/community/blob/master/README.md#ecosystem-projects

Reply:
Added that now. Thanks, I must have missed that part!